grails-app/controllers/org/broadinstitute/orsp/api/ReportController.groovy

```diff
@@ -21,24 +24,54 @@ class ReportController extends AuthenticatedController {
         render([error: "Not Implemented"] as JSON)
     }
 
-    def funding() {
+    def getFunding() {
+        Long i = 0
+        JSON.registerObjectMarshaller(Funding) {
+            def output = [:]
+            output['id'] = i++
+            output['type'] = it.issue.type
+            output['projectKey'] = it.issue.projectKey
+            output['protocol'] = it.issue.protocol
+            output['summary'] = it.issue.summary
+            output['status'] = it.issue.status
+            output['source'] = it.source
+            output['name'] = it.name
+            output['awardNumber'] = it.awardNumber
+            output['pis'] = getPIsDisplayName((Issue) it.issue)//it.issue.getPIs().collect{r -> getUser(r).displayName}
```

Comment:
remove unused code

Reply:
We've been putting these marshallers in a common place, `UtilityClass`

Reply:
Any other objects that you render as JSON which contain Funding objects will also have their fundings rendered as you designed, I think in the future it could be a bug. Follow registerIssueMarshaller as example

Reply:
Got it! I moved this to `UtilityClass`, followed `registerIssueMarshaller` as example, and removed unused code.